### ISSUE TITLE
How to create a session with one publisher and multiple subscribers 

### ISSUE THREAD

**Issue by zahrat** [opentok/opentok-react-native#717]
is there a way to create a live video broadcasting that has only one publisher and the subscribers can only see the publisher's video. something like live in instagram?

**Comment by beHaze** [comment]
@zahrat yes, you can programmatically achieve this. Your application can determine who publishes, see some of our documentation https://tokbox.com/developer/guides/broadcast/ https://github.com/opentok/broadcast-sample-app
